cvrf2csaf/cvrf2csaf.py

```diff
@@ -28,30 +42,85 @@ class DocumentHandler:
                                                     config['publisher_namespace'])
+        self.document_references = DocumentReferences()
+        self.document_source_lang = DocumentSourceLang()
+        self.document_title = DocumentTitle()
         self.document_tracking = DocumentTracking(config['cvrf2csaf_name'],
                                                   config['cvrf2csaf_version'],
                                                   config['force_update_revision_history'])
+        self.product_tree = ProductTree()
+        self.vulnerability = Vulnerability()
+
 
     def _parse(self, root):
         for elem in root.iterchildren():
             # get tag name without it's namespace, don't use elem.tag here
             tag = etree.QName(elem).localname
-            if tag == 'DocumentPublisher':
-                self.document_publisher.create_csaf(elem)
+
+            # ToDo: Lang and SourceLang are missing here.
+
+            if tag == 'Acknowledgments':
+                self.document_acknowledgments.create_csaf(root_element=elem)
+            elif tag == 'AggregateSeverity':
+                self.document_aggregate_severity.create_csaf(root_element=elem)
+            elif tag == 'DocumentType':
+                # Map DocumentType --> /document/category/
+                self.document_category.create_csaf(root_element=elem)
+            elif tag == 'DocumentDistribution':
+                self.document_distribution.create_csaf(root_element=elem)
+            elif tag == 'DocumentNotes':
+                self.document_notes.create_csaf(root_element=elem)
+            elif tag == 'DocumentPublisher':
+                self.document_publisher.create_csaf(root_element=elem)
+            elif tag == 'DocumentReferences':
+                self.document_references.create_csaf(root_element=elem)
+            elif tag == 'DocumentTitle':
+                self.document_title.create_csaf(root_element=elem)
             elif tag == 'DocumentTracking':
-                self.document_tracking.create_csaf(elem)
-            elif tag == 'ToDo':
-                # ToDo: Going through it tag by tag for further parsing
-                pass
+                self.document_tracking.create_csaf(root_element=elem)
+            elif tag == 'ProductTree':
+                self.product_tree.create_csaf(root_element=elem)
+            elif tag == 'Vulnerability':
+                self.vulnerability.create_csaf(root_element=elem)
+            elif tag in ['comment']:
+                self.comment.create_csaf(root_element=elem)
             else:
                 logging.warning(f'Not handled input tag {tag}. No parser available.')
 
     def _compose_final_csaf(self) -> dict:
+        # Merges first level leaves into final CSAF document.
+        # [mapping table](https://github.com/tschmidtb51/csaf/blob/csaf-2.0-what-is-new-table/notes/whats-new-csaf-v2.0-cn01.md#e4-mapped-elements)
+
         final_csaf = {'document': {}}
```

Comment:
Why not directly returning an anonymous literal dict (the method name is already a name)?

Like so:
```python
        return {
            'document': {
                  'acknowledgments': self.document_acknowledgments.csaf,
                  'aggregate_severity': self.document_aggregate_severity.csaf,
                  'category': self.document_category.csaf,
                  'distribution': self.document_distribution.csaf,
                  'lang': self.document_lang.csaf,
                  'notes': self.document_notes.csaf,
                  'publisher': self.document_publisher.csaf,
                  'source_lang': self.document_source_lang.csaf,
                  'references': self.document_references.csaf,
                  'title': self.document_title.csaf,
                  'tracking': self.document_tracking.csaf,
                  'references': self.document_references.csaf,
            },
            'product_tree': self.product_tree.csaf,
            'vulnerabilities': self.vulnerability.csaf,
        }
```